src/components/login/Login.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import { LOGIN_USER } from '../../redux/actions/actionTypes';
+
+const initialValues = {
+    email: 'RamRanch@gmail.com',
+    password: '********' 
```

Comment:
why are these the initial values? shouldn't they be the null value of each type which is `null` or `""`?